tests/func/test_checkout.py

```diff
@@ -569,3 +570,49 @@ def test(self):
             os.path.relpath(old_data_link, old_cache_dir),
             os.path.relpath(new_data_link, new_cache_dir),
         )
+
+
+@patch.object(
+    dvc.progress.CheckoutProgressCallback, "TOO_LONG_CHECKOUT_SECONDS", 0.0
+)
+class TestCheckoutShouldWarnOnDefaultCacheLinkType(TestDvc):
+    TOO_LONG_CHECKOUT_MSG = boxify(
+        "You can cut your checkout time considerably if you "
+        "will use different link type in your local cache. "
+        "Check:\n https://dvc.org/doc/commands-reference/"
+        "config#cache\n for more information.\n To disable "
+        "this message, run 'dvc config "
+        "cache.slow_checkout_warning False'",
+        border_color="blue",
+    )
+
+    def setUp(self):
+        super(TestCheckoutShouldWarnOnDefaultCacheLinkType, self).setUp()
+
+        ret = main(["config", "cache.type", "copy"])
```

Comment:
Again, default cache type is "reflink,copy", not "copy". You are testing this only for cases when user explicitly set "config cache.type copy".